test/unit/spec/appWidgets/input/select2ObjectinputSpec.js

```diff
@@ -101,6 +111,12 @@ define([
             defaultValue = 'apple',
             fakeServiceUrl = 'https://ci.kbase.us/services/fake_taxonomy_service';
 
+        const buildWidget = (objTypes) => {
+            testConfig = buildTestConfig(required, defaultValue, bus, objTypes);
+            widget = Select2ObjectInput.make(testConfig);
+            return widget;
```

Comment:
I don't know the context, but can you just return the output of Select2ObjectInput or does it need to be explicitly set to be `widget`?

Reply:
`widget` gets set as a variable under the `describe` block. I'm on the fence as to whether it needs to be, so I returned it for use in another test context.

Reply:
...and I never did actually use it that way. I'll remove the return line.